lib/handbook/addon/docs/components/bs-alert/template.md

```diff
@@ -1,9 +1,16 @@
 # bs-alert
 
-Generates a [boostrap alert](https://www.ember-bootstrap.com/api/classes/Components.Alert.html).
+Generates what is mostly a [boostrap alert](https://www.ember-bootstrap.com/api/classes/Components.Alert.html). 
+
+In addition to the standard look and functionality, we:
+ - Have a font-awesome close icon to match OSF styling;
+ - Have the ability to hide and show the alert.
```

Comment:
The ability to hide is actually something [ember-bootstrap has too](https://github.com/kaliber5/ember-bootstrap/blob/master/addon/templates/components/common/bs-alert.hbs#L1).

The other thing we do add, though, is [i18n](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/lib/osf-components/addon/components/bs-alert/template.hbs#L3).